src/components/__tests__/EmptyThought.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import App from '../../App'
+
+// eslint-disable-next-line fp/no-let
+let wrapper = null
+
+beforeAll(async () => {
+  wrapper = await window.createTestApp(<App />)
+})
+
+afterAll(() => {
+  window.cleanupTestApp()
+  wrapper = null
+})
+
+it('create and edit empty thought', async () => {
```

Comment:
Create and edit should be separate tests.

Reply:
Think about it, it's useless. In order to edit the thought, you need to create it, so why not test it? Same for delete.

Reply:
I disagree. If there is one test for creating a thought and for editing a thought, and the test fails after some changes are made in the future, we won't know whether there was a regression in creating a thought or editing a thought. While editing a thought will never work if creating a thought is broken, the inverse is not true. Having separate tests isolates these test cases and prevents this ambiguity from arising.

Reply:
But failing test tells you exact line where it broke, so we will know what exactly failed. If you still want me to have create and edit, no problem.

Reply:
Good point. Either one is a reasonable approach. Let's go with separate tests since that is my preference.